sdk/communication/communication-job-router-rest/src/azureCommunicationRoutingServiceClient.ts

```diff
@@ -2,23 +2,22 @@
 // Licensed under the MIT License.
 
 import type { ClientOptions } from "@azure-rest/core-client";
-import { getClient } from "@azure-rest/core-client";
 import type { KeyCredential, TokenCredential } from "@azure/core-auth";
 import { isTokenCredential } from "@azure/core-auth";
-import { logger } from "./logger";
-import type { AzureCommunicationRoutingServiceClient } from "./clientDefinitions";
+import type { AzureCommunicationRoutingServiceClient } from "../generated/clientDefinitions";
 import {
   createCommunicationAuthPolicy,
   isKeyCredential,
   parseClientArguments,
 } from "@azure/communication-common";
+import createClient from "../generated/azureCommunicationRoutingServiceClient";
 
 /**
  * Initialize a new instance of `AzureCommunicationRoutingServiceClient`
  * @param connectionString - The connectionString or url of your Communication Services resource.
  * @param options - the parameter for all optional parameters
  */
-export default function createClient(
+export default function createCustomizedClient(
```

Comment:
can we keep the createClient for this and change the generated import one as 
`import createClient as createGeneratedClient from "../generated/azureCommunicationRoutingServiceClient";`